deploy/staging/deployment.tpl.yml

```diff
@@ -21,22 +21,35 @@ spec:
         app: ${KUBE_NAMESPACE}
     spec:
       volumes:
-        - name: uploads
-          emptyDir: { }
         - name: php-socket
           emptyDir: { }
       terminationGracePeriodSeconds: 35
       serviceAccountName: ${KUBE_NAMESPACE}-service
+      securityContext:
+        runAsNonRoot: true
+        runAsUser: 101
+        runAsGroup: 101
+        seccompProfile:
+          type: "RuntimeDefault"
       containers:
         - name: nginx
           image: ${ECR_URL}:${IMAGE_TAG_NGINX}
           ports:
             - containerPort: 8080
           volumeMounts:
-            - name: uploads
-              mountPath: /var/www/html/public/app/uploads
```

Comment:
@EarthlingDavey is this okay to remove?

Reply:
Yes, that's right... it's safe to remove that volume mount because nginx doesn't need access to those files.